app/experimenter/docs/adrs/0001-create-new-nimbus-application.md

```diff
@@ -0,0 +1,119 @@
+  - Supporting both Normandy and Nimbus over time could cause unwanted complexity/duplication
+  - [Current Experimenter codebase may not align with FXA tools/processes/practices](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+  - Somewhat constricted by core designs decisions that were made in service of existing Normandy experimentation workflows that may not apply to Nimbus
+
+### Start A New Application In Python/Django
+
+Creating a new web application in Python/Django would allow us to reuse much of the existing code/infrastructure where applicable while clearly isolating the concerns of supporting existing experiment types and building out new Nimbus functionality.
+
+- Pros
+  - Start by cloning the repo and deleting all the unnecessary UI/DB/APIs
+  - Get a big head start by not needing to reimplement the reusable parts
+  - Isolate Nimbus concerns without fear of breaking Normandy experiments
+  - Consistent development and deployment workflow for both old and new applications
+  - Opportunity to rethink core code/architectural/tooling decisions where appropriate
+- Cons
+  - Lose single source of truth
+    - Possibly require people to know which application to refer to depending on experiment type
+    - Possibly create an API proxy in front of both to have a single endpoint for dependent callers
+    - More difficult to correlate experiments/learnings across the two applications
+  - Changes that must apply to both Normandy and Nimbus experiments must be made in two places while they're both being actively maintained
+  - Divergence in tooling/processes over time may create confusion when context switching between the two
+  - Possibly still somewhat constricted by past core design decisions
+  - [Python/Django may not align with FXA tools/processes](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+
+### Start A New Application in TypeScript/Node
+
+Creating a new web application in TypeScript/Node would allow us to completely restart our thinking about the entire structure of the application which would result in both code and workflow/UI that are purpose built from the ground up to solely address the concerns of Nimbus experiments.
+
+- Pros
+  - Totally greenfield approach allows fresh perspectives to flourish
+  - Stronger sense of collective ownership from all new contributors
+  - Every decision and line of code built specifically to address the needs of Nimbus
+  - TypeScript provides excellent feedback and guarantees which improves development experience
+  - Native async/websocket support allows richer UI experience when coupled with SPA frameworks (like React)
+  - [Fits more cohesively into the FXA team's tooling/processes](https://github.com/mozilla/fxa/blob/main/docs/adr/0020-application-architecture.md)
+  - Logic/types in TS can be easily shared across multiple projects potentially including Desktop/Mobile clients
```

Comment:
Because we already have [types written](https://github.com/mozilla/nimbus-shared/tree/main/types) for experiments that are in use (at least, for Desktop) this would be an immediate benefit